src/main/java/me/zeroeightsix/fiber/annotation/Setting.java

```diff
@@ -6,19 +6,58 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * Indicates that a field is a setting, and must be serialised as such.
```

Comment:
must be serialised as such or is serialised as such?

Reply:
Looking back at this - the second part is just confusing. It *is* being 'serialised as such', but of course also must be (by e.g. people writing custom serializers). It might be better to just write
> Indicates that a field is a setting.

What are your thoughts?

Reply:
If a setting is always serialised, and it seems that way, then leaving it out seems best yeah.